notebooks/basics/multiple-systems.ipynb

```diff
@@ -890,6 +952,8 @@
     "\\end{gathered}\n",
     "$$\n",
     "\n",
+    "That is, one imagines inserting a copy of the second matrix into every entry of the first, expanding out multiplications and removing the excess parentheses. As you can see, tensor products quickly increase the size of a matrix. As with vectors, this specific type of matrix multiplication is frequently called the *Kronecker product*. \n",
```

Comment:
We might accomplish all this better with a GIF. (Also don't really want to get into the distinction between the tensor and Kronecker products.)